src/components/organisms/Home.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-	<div class="home" style="background-color: black;">
+	<div class="home">
 		<homeImageContainer v-bind:imageContainer="imageContainer" v-bind:key="imageContainer.id" v-for="imageContainer in containers"/>
-		<homeArrowContainer/>
+		<HomeArrowContainer v-bind:containers="containers"/>
 	</div>
 </template>
 
 <script>
-	import homeArrowContainer from "../molecules/HomeArrowContainer.vue"
+	import HomeArrowContainer from "../molecules/HomeArrowContainer.vue"
```

Comment:
here is an example why organisms should have an a logic and not atoms
When i see this `Home.vue` organism I expect to get some logic right here because its an a organism, but since there no logic I expect here no slider functionality and this will cause a lot of problems researching where this functionality goes and I have no control over this component since its works like a lot of references passed to props
I suggest you to take a look of open source implementations of vue carousel-s so you can take a look on approaches of how it can be made